integrationtest/integration_test.go

```diff
@@ -148,18 +185,42 @@ func setupKubernetesAuth(t *testing.T, boundServiceAccountName string, mountConf
 		roleConfig = roleConfigOverride
 	}
 
-	_, err = client.Logical().Write("auth/kubernetes/role/test-role", roleConfig)
+	_, err := client.Logical().Write("auth/kubernetes/role/test-role", roleConfig)
 	if err != nil {
 		t.Fatal(err)
 	}
+}
 
-	return client, cleanup
+func setupKv1Mount(t *testing.T, client *api.Client, path string) func() {
```

Comment:
nit: Go casing conventions

```suggestion
func setupKVV1Mount(t *testing.T, client *api.Client, path string) func() {
```